contracts/.changeset/gold-countries-move.md

```diff
@@ -0,0 +1,10 @@
+---
+'@chainlink/contracts': patch
```

Comment:
This now needs to be created using `pnpm changeset:ccip` not `pnpm changeset` for all CCIP onchain PRs. That would put it in the right place and tag it with the right package